src/components/Switch/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useCallback } from 'react'
```

Comment:
why are you adding a new switch when obviously there is already one.
it also says so in the issue description IN BOLD.

Reply:
I've added this switch because there is no switch that acts as a simple toggle with just a knob, an area and animation. And the one used in StakeDeposit component is only reusable with copy-pasting, and isn't a component itself.

Reply:
Yes, that's exactly what I did. But Switch in StakeDeposit is just a styled-component and it has no logic inside. What I did is abstract the markup and logic into one component to make it more reusable. I can replace the Switch in StakeDeposit with my Switch if it's ok.

Reply:
whatever as long as there's only one switch everywhere